### PR TITLE
Geography page implementation

### PR DESCRIPTION
**Pull request by Gutts-n**
Related issue: https://github.com/transport-data/tdc-data-portal/issues/39

Created the map with this size because there is a problem with the width and zoom-out. if we add more width to the map and limit the maxZoom to 0.6 it breaks the canvas.

Another point is the gradient of the colors based on datasets, how will we implement it? Should we for every 193 (1650 / 9 tonalities of the cyan) dataset use a tonality of cyan?

### CODE REVIEW

File: frontend/src/pages/geography.tsx [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+  }, []);
+
+  return (
+    <>
+      <Head>
+        <title>Geography</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <Layout backgroundEffect>
```
Comment (luccasmmg) [inline frontend/src/pages/geography.tsx:151]: Try playing with the backgroundEffect (There is a prop to change the percentage) curreently you can see a slight difference in the color 
image

File: frontend/src/pages/geography.tsx [hunk 2]
```diff
@@ -0,0 +1,825 @@
...
+      maxzoom: 24,
+      "source-layer": "countries",
+    },
+    {
+      id: "countries-boundary",
+      type: "line",
+      paint: {
+        "line-color": "rgba(255, 255, 255, 1)",
+        "line-width": {
+          stops: [
```
Comment (luccasmmg) [inline frontend/src/pages/geography.tsx:252]: I think you can play around a little bit with these line-widths, i dont know if the other developers agree but on the mockup they seem smoother and i think thats in part cause of them being thicker
image
image

**Review by luccasmmg — APPROVED** [review]
Im going to approve, but if im honest i think the client idea doesnt really work when you consider a real world map that needs to deal with things like Russia and little islands, i think a globe projection would probably look better but that would involve redoing everything and we dont have the time

https://docs.mapbox.com/mapbox-gl-js/example/add-image-generated/

**Review by willy1989cv — CHANGES_REQUESTED** [review]
- [ ] Tooltip
The flag should be full rounded as per figma
image

- [ ] Map width should be the whole container width
image

- [ ] Cards on Mobile should be 100% width
image

**Comment by Gutts-n** [comment]
> * [ ]  Tooltip
>   The flag should be full rounded as per figma
>   image

@willy1989cv the API that I'm using to get the flags has images sliced in height, take a look at this endpoint https://flagsapi.com/US/flat/64.png, do you know some property to add to the img to stretch the image?

**Comment by willy1989cv** [comment]
>  do you know some property to add to the img to stretch the image?

Try using object-fit. I tried in console, and used object-fit: none, it seems to work

**Review by willy1989cv — APPROVED** [review]
LGTM. the only thing I am not sure is. the countries with no datasets, should they show the tooltip as well?

**Comment by Gutts-n** [comment]
> LGTM. the only thing I am not sure is. the countries with no datasets, should they show the tooltip as well?

Idk as well, I'll leave a comment to Uche and Osahon to check it

**Pull request merged**